.eslintrc

```diff
@@ -2,6 +2,9 @@ parser: babel-eslint
 extends:
   - jason/react
   - prettier
+  - plugin:react-hooks/recommended
```

Comment:
you've got this over specified here. If we add the rules manually like you have below the extends isn't needed here